test/GCClassic/intTestCreate.sh

```diff
@@ -31,444 +27,222 @@
+# Change to the directory where we will create the rundirs
+printf "\nCreating new run directories:\n"
 
 #=============================================================================
 # Create individual run directories: 4x5 - MERRA2 - 72L
 #=============================================================================
 
-dir="gc_4x5_CH4_merra2"
-create_rundir "3\n1\n1\n1\n${root}\n${dir}\nn\n"          ${root} ${dir} ${log}
-
-dir="gc_4x5_aerosol_merra2"
-create_rundir "2\n1\n1\n1\n${root}\n${dir}\nn\n"          ${root} ${dir} ${log}
-
-dir="gc_4x5_fullchem_merra2"
-create_rundir "1\n1\n1\n1\n1\n${root}\n${dir}\nn\n"       ${root} ${dir} ${log}
-
-dir="gc_4x5_fullchem_merra2_LuoWd"
-create_rundir "1\n1\n1\n1\n1\n${root}\n${dir}\nn\n"       ${root} ${dir} ${log}
-
-dir="gc_4x5_fullchem_aciduptake_merra2"
-create_rundir "1\n5\n1\n1\n1\n${root}\n${dir}\nn\n"       ${root} ${dir} ${log}
-
-dir="gc_4x5_fullchem_APM_merra2"
-create_rundir "1\n7\n1\n1\n1\n${root}\n${dir}\nn\n"       ${root} ${dir} ${log}
-
-dir="gc_4x5_fullchem_benchmark_merra2"
-create_rundir "1\n2\n1\n1\n1\n${root}\n${dir}\nn\n"       ${root} ${dir} ${log}
-
-dir="gc_4x5_fullchem_complexSOA_merra2"
-create_rundir "1\n3\n1\n1\n1\n1\n${root}\n${dir}\nn\n"    ${root} ${dir} ${log}
-
-dir="gc_4x5_fullchem_complexSOA_SVPOA_merra2"
-create_rundir "1\n3\n2\n1\n1\n1\n${root}\n${dir}\nn\n"    ${root} ${dir} ${log}
-
-dir="gc_4x5_fullchem_marinePOA_merra2"
-create_rundir "1\n4\n1\n1\n1\n${root}\n${dir}\nn\n"       ${root} ${dir} ${log}
-
-dir="gc_4x5_fullchem_RRTMG_merra2"
-create_rundir "1\n8\n1\n1\n1\n${root}\n${dir}\nn\n"       ${root} ${dir} ${log}
-
-dir="gc_4x5_fullchem_TOMAS15_merra2_47L"
-create_rundir "1\n6\n1\n1\n1\n2\n${root}\n${dir}\nn\n"    ${root} ${dir} ${log}
+# 4x5 merra2 CH4
+create_rundir "3\n1\n1\n1\n${itRoot}\n\nn\n" "${log}" "${itRoot}"
 
-dir="gc_4x5_fullchem_TOMAS40_merra2_47L"
-create_rundir "1\n6\n2\n1\n1\n2\n${root}\n${dir}\nn\n"    ${root} ${dir} ${log}
+# 4x5 merra2 CO2
+create_rundir "4\n1\n1\n1\n${itRoot}\n\nn\n" "${log}" "${itRoot}"
 
-dir="gc_4x5_Hg_merra2"
-create_rundir "5\n1\n1\n1\n${root}\n${dir}\nn\n"          ${root} ${dir} ${log}
+# 4x5 merra2 aerosol
+create_rundir "2\n1\n1\n1\n${itRoot}\n\nn\n" "${log}" "${itRoot}"
 
-dir="gc_4x5_POPs_BaP_merra2"
-create_rundir "6\n1\n1\n1\n1\n${root}\n${dir}\nn\n"       ${root} ${dir} ${log}
+# 4x5 merra2 carbon
+create_rundir "12\n1\n1\n1\n${itRoot}\n\nn\n" "${log}" "${itRoot}"
 
-dir="gc_4x5_tagCH4_merra2"
-create_rundir "7\n1\n1\n1\n${root}\n${dir}\nn\n"          ${root} ${dir} ${log}
-
-dir="gc_4x5_tagCO_merra2"
-create_rundir "8\n1\n1\n1\n${root}\n${dir}\nn\n"          ${root} ${dir} ${log}
+# DEBUG: Exit after creating a couple of rundirs if $quick is "yes"
+if [[ "x${quick}" == "xyes" ]]; then
+    cd ${thisDir}
+    exit 0
+fi
 
-dir="gc_4x5_tagO3_merra2"
-create_rundir "9\n1\n1\n1\n${root}\n${dir}\nn\n"          ${root} ${dir} ${log}
+# 4x5 merra2 fullchem
```

Comment:
Should we move at least one fullchem simulation to the quick test section?

Reply:
This is now done in commit 9f68416bf630f7e303e376f1868a0bd7121bcf53